src/nri/metricsProcesor.go

```diff
@@ -143,6 +171,10 @@ func processMetricGauge(metricFamily dto.MetricFamily, entityRules EntityRules,
 func addMetadata(metadata metadataMap, e *integration.Entity) {
 	var err error
 	for k, v := range metadata {
+		// exporter sends service_name in camel case we need to convert it to lowercase
+		if k == "service_name" {
+			v = strings.ToLower(v)
+		}
```

Comment:
Is this something that changed in recent versions of the exporters? Is it possible to update/extend tests to cover it? Eg: using a camel-cased service-name.